app/views/shop/products/_form.html.haml

```diff
@@ -1,7 +1,35 @@
 %form{action: main_app.cart_path}
   %products{"ng-controller" => "ProductsCtrl", "ng-init" => "refreshStaleData()", "ng-show" => "order_cycle.order_cycle_id != null", "ng-cloak" => true }
 
-    = render partial: "shop/products/searchbar"
+    // TODO: Needs an ng-show to slide content down
```

Comment:
this code is not part of this PR and needs to be removed.
you can fetch latest master and rebase this PR. Locally this would look like this:
git checkout upstream master
git pull upstream master
git checkout 4534-mobile-searchkey
git rebase master
-- resolve conflicts if any
git push -f origin 4534-mobile-searchkey

afterwards, make sure only your changes are included in the PR here:
https://github.com/openfoodfoundation/openfoodnetwork/pull/5356/files

Reply:
Ahh, okay. I want to make sure I'm on the same page so to speak. The conflict was with the `render partial: "shop/products/searchbar" line, which I removed. Are you referring to the comment line? Currently that section in my code looks like this...
```
    .row
      .small-12.medium-6.large-5.columns
        %input{type: "search",
        "ng-model" => "search",
        placeholder: t(:products_search),
        "ng-debounce" => "200",
        inputmode: "search",
        name: "search",
        "ofn-disable-enter" => true}

      .small-12.medium-6.large-6.large-offset-1.columns
        = render partial: "shop/products/filters"

    .row
```

Reply:
> Always have a look at /files in your PR https://github.com/openfoodfoundation/openfoodnetwork/pull/5356/files
> You are adding some indentation to an unrelated view and indentation matters. That's why the build is breaking all over with:
> 
> ```
> 37) As a consumer I want to shop with a distributor Viewing a distributor when shopping requires a customer when logged in as the owner shows just products
>      Failure/Error: = render partial: "shop/products/searchbar"
>      
>      ActionView::Template::Error:
>        The line was indented 2 levels deeper than the previous line.
>      # ./app/views/shop/products/_form.html.haml:4
> ```

Thank you!